_posts/05-03-01-Date-and-Time.md

```diff
@@ -4,15 +4,12 @@ isChild: true
 anchor:  date_and_time
 ---
 
-## Date and Time {#date_and_time_title}
+## ডেট এন্ড টাইম {#date_and_time_title}
 
-PHP has a class named DateTime to help you when reading, writing, comparing or calculating with date and time. There
-are many date and time related functions in PHP besides DateTime, but it provides nice object-oriented interface to
-most common uses. It can handle time zones, but that is outside this short introduction.
+PHP তে DateTime নামে একটি ক্লাস আছে যা আপনাকে ডেট এবং টাইম এর ইনপুট, আউটপুট, তুলনা করা অথবা অন্যান্য হিসাব-নিকাশ এর কাজে সাহায্য করবে। যদিও PHP তে DateTime ক্লাস ছাড়াও ডেট এবং টাইম সম্পর্কিত অন্যান্য অনেক ফাংশন রয়েছে, কিন্তু সাধারণভাবে ব্যবহারের ক্ষেত্রে এটি মানসম্মত অবজেক্ট ওরিয়েন্টেড ইন্টারফেস প্রদান করে। এই ক্লাস টাইম জোন নিয়েও কাজ করতে পারে, তবে এখানে তা বিস্তারিতভাবে উল্লেখ করা হলো না।
```

Comment:
... যা আপনাকে **তারিখ ও সময়ের** ইনপুট, আউটপুট, তুলনা করা ...

... PHP তে DateTime ক্লাস ছাড়াও **তারিখ ও সময়** সম্পর্কিত ...

Reply:
@arobd vai, "তারিখ ও সময়" না লিখে "ডেট এবং টাইম" রেখে দেওয়াই যুক্তিসঙ্গত মনে হচ্ছিল।

Reply:
This are not a programming terms, we should translate these.